www/static/rss.svg

```diff
@@ -1,10 +1,10 @@
 <!-- in Delapouite we trust: https://game-icons.net/1x1/delapouite/rss.html -->
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
   <style>
-    path { fill: #222; }
+    path { fill: #F56960; }
     :root { color-scheme: light dark; }
     @media (prefers-color-scheme: dark) {
-      path { fill: #ccc; }
+      path { fill: #F56960; }
     }
```

Comment:
If using the same color for both backgrounds is intended, the whole `<style>` section could be replaced by a `fill` attribute directly in the `<path>` element.